### ISSUE TITLE
ACCOUNT MGMT

### ISSUE THREAD

**Issue by macy125** [wgimson/QRdly#46]
https://serene-everglades-65051.herokuapp.com/account

We don't need e-mail verification because they'll be registering

**Comment by wgimson** [comment]
@macy125 @alexgimson86 

This is waiting on a detailed workflow for customer onboarding.

**Comment by wgimson** [comment]
I believe this is an old duplicate of https://github.com/wgimson/QRdly/issues/54

If so we should close this one.
